soh/include/z64.h

```diff
@@ -2245,6 +2245,8 @@ typedef enum {
     LED_SOURCE_TUNIC_ORIGINAL,
     LED_SOURCE_TUNIC_COSMETICS,
     LED_SOURCE_HEALTH,
+    //LED_SOURCE_NAVI_ORIGINAL,
+    LED_SOURCE_NAVI,
```

Comment:
I intend to go back and make it so Navi can either be the original colours or the Cosmetics Editor colours.